packages/@okta/vuepress-site/docs/reference/api-overview/index.md

```diff
@@ -79,7 +79,57 @@ Okta will derive client request context directly from the HTTP request headers a
 
 Okta supports the standard `User-Agent` HTTP header to identify the user's browser or application. Always send a `User-Agent` string to uniquely identify your client application and version such as `Oktaprise/1.1`.
 
-> If your application is acting as a gateway or proxy, you should forward the `User-Agent` of the originating client with your API requests.
+> **Note:** If your application is acting as a gateway or proxy, you should forward the `User-Agent` of the originating client with your API requests.
+
+### Format a User-Agent string
+
+Use the following examples to make sure that the User-Agent string that your app constructs is in the correct format. This ensures that Okta can parse the OS and Browser fields:
+
+Examples of User-Agent strings that can be correctly parsed by Okta:
+
+`Mozilla/5.0 (iPhone; CPU iPhone OS 13_3_1 like Mac OS X) AppleWebKit/534.46 (KHTML, like Gecko) Version/13.3.1 Mobile/9B179 Safari/7534.48.3 OktaMobile/6.3.0`
+
+`Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 10.0; WOW64; Trident/7.0; .NET4.0C; .NET4.0E; .NET CLR 2.0.50727; .NET CLR 3.0.30729; .NET CLR 3.5.30729; Zoom 3.6.0)`
+
+`Mozilla/5.0 (Linux; Android 10; SM-G9600 Build/QP1A.190711.020; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/80.0.3987.119 Mobile Safari/537.36 OktaMobile/4.5.1`
+
+Examples of User-Agent strings that can't be correctly parsed by Okta:
+
+`iPhone/iPhone10,4/13.3.1/2020.2.6480/Owners iPhone`
+
+`LGE/LG-Q710AL/8.1.0/2020.2.5794/22025912-357a-40ef-bfd9-62df3b8d08fe`
+
+`motorola/moto e5 plus/8.0.0/2020.2.5794/2b57f5c4-0b17-4b14-b933-2ec5c125b124`
+
+#### Different formatting approaches
+
+We recommend that you use a template like the following to format the User-Agent string:
+
+`User-Agent: Mozilla/5.0 (<system-information>) <platform> (<platform-details>) <extensions>`
+
+Alternatively, you can make sure that the following string values are present in the User-Agent string so that the OS and browser are detected:
+
+**iOS**
+
+- The presence of the words `apple` or `ios` and at least one of these values: `iphone`, `ipad`, `ipod`, `ipad`
+
+  **Example:** `iPhone/ios/iPhone10,4/13.3.1/2020.2.6480/Owners iPhone`
```

Comment:
Again, I think these examples are from a customer and we should remove them. We should just include the note about the presence of certain strings to make sure the OS and Browser show up correctly for IOS and Android.

Reply:
@rpamidimarri-okta - @lilyhsiao-okta is requesting that we add some examples here. Can I get an iOS and an Android GENERIC example to include, please?

Reply:
I would not recommend adding examples. The issue with examples is that people would literally copy and paste into their code which we do not want.

Reply:
I agree with @srinivasanagandla-okta . There are so many ways a UA string can be constructed that we cannot show very generic examples and we dont want customers to copy the ones from the doc. The https://oktawiki.atlassian.net/wiki/spaces/eng/pages/900764066/Technical+Questions+from+Bank+Of+Oklahoma talks about specific examples based on the customer samples.. so, those examples make sense for that particular customer.

Reply:
The reason for suggesting examples here is the sentence above it:

"you can make sure that certain string values are present in the User-Agent string so that the OS and Browser are detected" 

This makes it sound like the string values can be any where and it would be parsable. If that's not the case then we could re-word the sentence. 
That's where an example would be helpful.